test-apps/ui-test-app/src/frontend/index.tsx

```diff
@@ -73,6 +73,7 @@ import { IModelIndexFrontstage } from "./appui/frontstages/IModelIndexFrontstage
 import { SignInFrontstage } from "./appui/frontstages/SignInFrontstage";
 import { BrowserRouter, Route, Switch } from "react-router-dom";
 import { InspectUiItemInfoTool } from "./tools/InspectTool";
+import { MichelTestPrefs } from "./MichelTestPrefs";

```

Comment:
Looks like leftover code as it is a new import that is not used anywhere.

Reply:
ooops, obviously that shouldnt be there